backend/inspirehep/submissions/views.py

```diff
@@ -125,13 +125,43 @@ def post(self):
         data = conference_loader_v1()
         record = ConferencesRecord.create(data)
         db.session.commit()
-        # TODO: if not is_superuser_or_cataloger_logged_in(): self.create_ticket(record, "rt/new_conferences.html")
-        return jsonify(
-            {
-                "pid_value": record["control_number"],
-                "cnum": record.get("cnum"),
-            }
-        ), 201
+        if not is_superuser_or_cataloger_logged_in():
+            self.create_ticket(record, "rt/new_conference.html")
+        return (
+            jsonify(
+                {"pid_value": record["control_number"], "cnum": record.get("cnum")}
+            ),
+            201,
+        )
+
+    def create_ticket(self, record, rt_template):
```

Comment:
this function is very similar to https://github.com/inspirehep/inspirehep/blob/1ae762f48e218debf3ab96062cdd2b3c828f91e6/backend/inspirehep/submissions/views.py#L293 can we move it to BaseSubmissionsResource?